multidict/_pair_list.c

```diff
@@ -994,6 +992,13 @@ pair_list_update(PyObject *op1, PyObject *op2)
 }
 
 
+PyObject *
+pair_list_calc_identity(PyObject *op, PyObject *key)
```

Comment:
Ok with the method as a part of the transition.
In the end, we probably will not need this function, at least not in public API.

Reply:
Yes. This is just for a transition.